README.md

```diff
@@ -40,3 +41,133 @@ See also prior version [aurora-cli](https://github.com/aurora-is-near/aurora-cli
     *`cargo install --git https://github.com/aurora-is-near/aurora-cli-rs.git`* 
 - 🔍 Check out what each command is for in the official Aurora [docs](https://doc.aurora.dev/tools/aurora-cli)
 - ✋ Have questions? Ask them at the official Aurora [forum](https://forum.aurora.dev/)
+
+## Example of usage
+
+In the following example, we will see how to deploy Aurora EVM on the `localnet`. Also, we will deploy simple EVM smart
+contract and will be interacting with it.
+
+#### What we need to have
+- Rust 1.68.0 or newer
+- Python3
+
+First what we need to do is to install `aurora-cli`:
+
+```shell
+git clone https://github.com/aurora-engine/aurora-cli-rs
+cd aurora-cli-rs && cargo install --path . 
+```
+
+Next we need to start a NEAR node locally. For that we will be using the utility by NEAR team `nearup`.
+
+Install `nearup`:
+```shell
+pip3 install --user nearup
+```
+
+Start NEAR node:
+```shell
+nearup run localnet --home /tmp/localnet
+```
+
+After that we need to prepare an account and create a file with private key for the Aurora EVM and .
+
+```shell
+aurora-cli --near-key-path /tmp/localnet/node0/validator_key.json create-account \
+  --account aurora.node0 --balance 100 > /tmp/localnet/auora_key.json
+```
+
+Let's check if the account has been created successfully:
+```shell
+aurora-cli view-account aurora.node0
+```
+
+If everything went well, the response should be like this:
+```json
+{
+  "amount": "100000000000000000000000000",
+  "locked": "0",
+  "code_hash": "11111111111111111111111111111111",
+  "storage_usage": 182,
+  "storage_paid_at": 0
+}
+```
+
+Now we can deploy and initialize Aurora EVM. But first we should download the latest version of Aurora EVM.
+```shell
+curl -sL https://github.com/aurora-is-near/aurora-engine/releases/download/latest/aurora-mainnet.wasm -o /tmp/aurora-mainnet.wasm
+```
+
+Deploy Aurora EVM:
+```shell
+aurora-cli --near-key-path /tmp/localnet/auora_key.json deploy-aurora /tmp/aurora-mainnet.wasm
+```
+
+Initialize Aurora EVM:
+```shell
+aurora-cli --engine aurora.node0 --near-key-path /tmp/localnet/auora_key.json init --chain-id 1313161556 --owner-id aurora.node0
+```
+
+And now we can deploy EVM smart contract. In our example, it will be a simple counter that can return its current value
+and increment and decrement its value.
+
+But before we need to generate private key to have possibility to sign transactions:
+```shell
+aurora-cli key-pair --random
+```
+The response should be similar to this:
+```json
+{
+  "address": "0xa003a6e0e1a1dc40aa9e496c1c058b2667c409f5",
+  "secret_key": "3fac6dca1c6fc056b971a4e9090afbbfbdf3bc443e9cda595facb653cb1c01e1"
+}
+```
+**_NOTE:_** The key could be used for the demonstration only. 
+
+Deploy EVM smart contract:
+```shell
+aurora-cli --engine aurora.node0 --near-key-path /tmp/localnet/auora_key.json deploy \
```

Comment:
One thing I'm a bit worried about if that people may store an actual key they intend to use at the `tmp` path. I get the warning above, but hopefully may not cause issues for some. What -may- be better is to write an example script, OR a Rust example which runs commands?